### PR TITLE
refactored according to the last review comments

### PR DESCRIPTION
**Pull request by maksimk99**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -161,7 +161,7 @@
         <artifactId>maven-checkstyle-plugin</artifactId>
         <version>${maven.checkstyle.plugin}</version>
         <configuration>
-          <configLocation>my_checkstyle.xml</configLocation>
+          <configLocation>checkstyle.xml</configLocation>
```
Comment (aleksieiev) [inline pom.xml]: Do you really need qualify the name? It's default name ;-)

File: src/main/java/com/spring/rest/dao/CarCharacteristicsDao.java [hunk 2]
```diff
@@ -6,6 +6,8 @@
 
 /**
  * The interface provides methods to manage CarCharacteristics model.
+ * {@see CarCharackteristics}
```
Comment (aleksieiev) [inline src/main/java/com/spring/rest/dao/CarCharacteristicsDao.java]: Don't believe, its wrong

File: src/main/java/com/spring/rest/dao/impl/CarCharacteristicsDaoImpl.java [hunk 3]
```diff
@@ -29,10 +30,10 @@
     @Value("${car.characteristics.get}")
     private String GET_CAR_CHARACTERISTICS_SQL;
     /**
-     * SQL query to get cars characteristics.
+     * SQL query to get list of cars characteristics.
      */
-    @Value("${cars.characteristics.get}")
-    private String GET_CARS_CHARACTERISTICS_SQL;
+    @Value("${list.cars.characteristics.get}")
```
Comment (aleksieiev) [inline src/main/java/com/spring/rest/dao/impl/CarCharacteristicsDaoImpl.java]: `car.characteristics.list.get` is better as for me.

File: src/main/java/com/spring/rest/dao/impl/CarFeatureDaoImpl.java [hunk 4]
```diff
@@ -27,10 +28,10 @@
     @Value("${car.feature.get}")
     private String GET_CAR_FEATURE_SQL;
     /**
-     * SQL query to get car features.
+     * SQL query to get list of car features.
      */
-    @Value("${car.features.get}")
-    private String GET_CAR_FEATURES_SQL;
+    @Value("${list.car.features.get}")
```
Comment (aleksieiev) [inline src/main/java/com/spring/rest/dao/impl/CarFeatureDaoImpl.java]: the same like above

File: src/main/java/com/spring/rest/dao/impl/CarMakeDaoImpl.java [hunk 5]
```diff
@@ -25,10 +26,10 @@
     @Value("${car.make.get}")
     private String GET_CAR_MAKE_SQL;
     /**
-     * SQL query to get car makes.
+     * SQL query to get list of car makes.
      */
-    @Value("${car.makes.get}")
-    private String GET_CARS_MAKES_SQL;
+    @Value("${list.car.makes.get}")
```
Comment (aleksieiev) [inline src/main/java/com/spring/rest/dao/impl/CarMakeDaoImpl.java]: the same above

**Review by aleksieiev — COMMENTED** [review]
Try again, bro

File: src/main/java/com/spring/rest/dao/impl/CarCharacteristicsDaoImpl.java (hunk 3 above)
Comment (maksimk99) [reply src/main/java/com/spring/rest/dao/impl/CarCharacteristicsDaoImpl.java]: should I write constants in the same order as a property name?

**Comment by maksimk99** [comment]
make some changes
also, add model hierarchy and connections between them, see in docs folder

**Review by aleksieiev — APPROVED** [review]

**Pull request merged**